examples/onft721/package.json

```diff
@@ -11,7 +11,7 @@
     "lint:fix": "eslint --fix '**/*.{js,ts,json}' && prettier --write . && solhint 'contracts/**/*.sol' --fix --noPrompt",
     "lint:js": "eslint '**/*.{js,ts,json}' && prettier --check .",
     "lint:sol": "solhint 'contracts/**/*.sol'",
-    "test": "$npm_execpath run test:forge && $npm_execpath run test:hardhat",
+    "test": "echo 'No tests yet'",
```

Comment:
yeah i saw this fail locally... was the cache causing CI to pass?

Reply:
yeah the last time we updated the base image was 8 or so months ago and apparently since then `forge test` started failing if it didn't find any tests